package.json

```diff
@@ -4,7 +4,7 @@
   "description": "This is a project for learning Node/Express. In fact it will be proxy server for original NASA API.",
   "main": "index.js",
   "scripts": {
-    "start": "node index",
+    "dev": "nodemon -r dotenv/config ./server.js ",
```

Comment:
I think you can remove nodemon and use internal node functionality 
```
"dev": "node --watch ./server.js",
```
If you want to use env variables from env file:
```
"dev": "node --watch ./server.js --env-file=.env"
```